src/rgw/rgw_bucket.cc

```diff
@@ -55,7 +55,8 @@
 #define dout_context g_ceph_context
 #define dout_subsys ceph_subsys_rgw
 
-#define BUCKET_TAG_TIMEOUT 30
+// seconds for timeout during RGWBucket::check_object_index
+constexpr uint64_t BUCKET_TAG_QUICK_TIMEOUT = 30;
```

Comment:
That's used during a "radosgw-admin bucket check ..." (don't think "--fix" is required). It's trying to nail the bucket index so it can calc stats and report accurately it seems.